### ISSUE TITLE
Application error

### ISSUE THREAD

**Issue by MaxGhenis** [compute-tooling/compute-studio#151]
In running https://www.compmodels.org/PSLmodels/Tax-Brain/41142 (a CPS-based reform eliminating the payroll tax cap), Comp hung at "Estimated 2 minutes remaining" for several minutes. When I refreshed the page, I got this:
![image](https://user-images.githubusercontent.com/6076111/59155648-6137ce80-8a43-11e9-95f8-af9175a1d959.png)

And now compmodels.org shows that error on the homepage and for all compmodels pages.

**Comment by hdoupe** [comment]
Hey Max, I just restarted the cluster and the webapp. Things look like they are back to normal now. I ran one of your failed simulations ([PSLmodels/Tax-Brain/41142](https://www.compmodels.org/PSLmodels/Tax-Brain/41142/edit/)) and it completed successfully: https://compmodels.org/PSLmodels/Tax-Brain/41145/.

I scaled up the servers for a demo with some students last week, and my hunch is that docker swarm didn't get the memo that the extra servers were no longer online. I'll continue to investigate this. The good news is that the COMP worker infrastructure is about to get a [whole lot better](https://github.com/comp-org/comp-ce/pull/147).

Thanks again for using COMP early on. I'm working to make it a better experience, and in the mean time, I really appreciate your patience and bug reports.
